package.json

```diff
@@ -111,14 +111,17 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "tsc -p ./",
-    "compile": "tsc -watch -p ./",
-    "preinstall": "node ./scripts/preinstall.js",
+    "vscode:prepublish": "npm run compile",
+    "compile": "tsc -p ./",
+    "watch": "tsc -watch -p ./",
+    "preinstall": "npm i --save mvn-artifact-download && node ./scripts/preinstall.js",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "node ./node_modules/vscode/bin/test",
-    "tslint": "gulp tslint"
+    "tslint": "gulp tslint",
+    "ui-test": "npm run compile && extest setup-and-run --code_version 1.38.1 out/ui-test/*.js -o src/ui-test/settings.json"
```

Comment:
I'm wondering how/when we will update this version

Reply:
for now it would be manual decision after local verification that UI tests are working